updates in contact us ,events and webinar ,internship ,jobs page

Pratyush-who opened the pull request:
made changes in ui of the following pages

Aarzookhunger commented:
@Pratyush-who which issue does this PR fix and attach supporting screenshots regarding what changes have u made.

Pratyush-who commented:
> @Pratyush-who which issue does this PR fix and attach supporting screenshots regarding what changes have u made.
updated the following ui's and made the add event button working... for the events page... and also fixed other minor UI related and favicon issues

![image](https://github.com/user-attachments/assets/bbf2907b-dffc-42fe-a872-9f4c6cf6511c)
![image](https://github.com/user-attachments/assets/f45b66af-b8a0-48ad-8b10-a28647c45dfd)
![image](https://github.com/user-attachments/assets/4de755e6-39b9-4e98-b708-616f893babec)
![image](https://github.com/user-attachments/assets/49015542-ab42-42c6-9810-794463c49378)
![image](https://github.com/user-attachments/assets/121f56d3-6441-4409-9268-886f337d18f3)
![image](https://github.com/user-attachments/assets/17004a49-04df-4c60-8afb-87c2b0d346ee)
![image](https://github.com/user-attachments/assets/2c872d71-56ae-4be6-878a-30afcb80cccc)

Aarzookhunger commented:
@Pratyush-who mention issue which you were assigned to fix.

Pratyush-who commented:
Feature: Add Event/Webinar #123
Jobs page UI #161
enhancement: UI #64
Redesign jobs section #15

I wasn't assigned any... I just forked and raised the pr doing things I liked... It was my first open source... So maybe I did something wrong... Idk... But the above mentioned ones can be the fixes I made...

Mayureshd-18 reviewed (changes requested):
@Pratyush-who Please resolve the conflicts and first raise an issue and then only create a PR.
